src/contracts/BPool.sol

```diff
@@ -15,7 +15,7 @@ contract BPool is BToken, BMath, IBPool {
   using SafeERC20 for IERC20;
   /// @dev BFactory address to push token exitFee to
 
-  address internal _factory;
+  address internal immutable _FACTORY;
```

Comment:
toughts on making it public? given there are going to be many different factories, other contracts might want to know that at execution time 
(and it'd enable us to get rid of the manual mock)

Reply:
wdyt if we add it on a different PR, since this one is just for gas optimizations?